includes/class-wc-payments-checkout.php

```diff
@@ -145,14 +145,8 @@ public function register_scripts_for_zero_order_total() {
 			! has_block( 'woocommerce/checkout' )
 		) {
 			WC_Payments::get_gateway()->tokenization_script();
-			$script_handle = 'WCPAY_CHECKOUT';
```

Comment:
Since `WCPAY_CHECKOUT` is not used here anymore, do you think its references can also be removed from anywhere else in the application? 👀

Reply:
That's a good suggestion. The main reason I kept this constant for now is that removing it will probably lead to deleting the `WC_Payments_Checkout` class because it is used as the one & only while registering scripts or providing payment fields to JS. 

`WCPAY_CHECKOUT` is also used in Subscriptions for some reason, and I'd like to check it more deeply on why none of the UPE constants is used there before removing it.

I'd probably proceed with removing it within one of the next PRs, if all the above makes sense.

Reply:
[Here's where it gets registered](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payments-checkout.php#L130) and I'd like to start the cleanup of the `WCPAY_CHECKOUT` from there. Probably as part of the issue to streamline `class-wc-payments-checkout.php` & `class-wc-payments-upe-checkout.php`. Let me know how it sounds for you and I'm happy to align further 👍